lib/phonebook/ui/pages/association_editor_page/association_editor_page.dart

```diff
@@ -183,8 +181,6 @@ class AssociationEditorPage extends HookConsumerWidget {
                                         } else if (value.isEmpty) {
```

Comment:
Those two condition are returning the same message, can you just write a single one

Reply:
Fixed in next commit